src/btree.c

```diff
@@ -758,8 +764,9 @@ btree_build_leaf_splitting_plan(const btree_config          *cfg, // IN
                                 const leaf_incorporate_spec *spec) // IN
 {
    /* Split the content by bytes -- roughly half the bytes go to the
-      right node.  So count the bytes, including the new entry to be
-      inserted. */
+    * right node.  So count the bytes, including the new entry to be
```

Comment:
Most changes are like this one, to update  multi-line comment blocks to conform to this style:

```
/*
 * Start of multi-line comment.
 * Continuation line ..
 */
```

As the previous author did not have `/*` on its own line for most / all multi-line comment blocks in this file, I left L766 (start line of comment block) unchanged.